stablehlo/reference/Element.cpp

```diff
@@ -174,6 +174,33 @@ bool areApproximatelyEqual(APFloat f, APFloat g) {
   return std::fabs(f.convertToDouble() - g.convertToDouble()) <= 0.0001;
 }
 
+bool areClose(APFloat f, APFloat g, APFloat abs_error) {
```

Comment:
Can't we merge `areClose` and `areApproximatelyEqual` into a single function with signature

```cc
bool areClose(APFloat f, APFloat g, APFloat, APFloat abs_error)
``` 

Depending on where it is called from the `abs_error` can be default `0.0001` or explictly specified?

Reply:
yes, will do it.

Reply:
Added optional `abs_error` argument to `areApproximatelyEqual` and eliminated `areClose`.